src/sparseml/transformers/utils/sparse_model.py

```diff
@@ -473,3 +473,26 @@ def get_shared_tokenizer_src(student: Module, teacher: Optional[Module]) -> str:
     else:
         src_model = student
     return src_model.config._name_or_path
+
+
+def download_zoo_training_dir(zoo_stub: str) -> str:
```

Comment:
let's potentially make it a more high level function, it solves potentially more irritating use cases.

Reply:
Moved to sparseml.utils